src/controllers/Onboarding/file.js

```
@@ -0,0 +1,118 @@
+const db = require("../../../models");
+require("dotenv").config();
+const message = require("../../../constants/messages.json");
+
+const convertFileToBase64 = (document) => {
+    document.file = document.file && document.file.toString("base64");
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)